src/PoshGitTypes.ps1

```diff
@@ -244,19 +244,20 @@ class PoshGitPromptSettings {
     [Nullable[bool]]$EnableFileStatusFromCache        = $null
     [string[]]$RepositoriesInWhichToDisableFileStatus = @()
 
-    [string]$DescribeStyle   = ''
-    [psobject]$WindowTitle   = {param($GitStatus, [bool]$IsAdmin) "$(if ($IsAdmin) {'Administrator: '})$(if ($GitStatus) {"posh~git ~ $($GitStatus.RepoName) [$($GitStatus.Branch)] ~ "})PowerShell $($PSVersionTable.PSVersion) ($PID)"}
+    [string]$DescribeStyle = ''
+    [psobject]$WindowTitle = {param($GitStatus, [bool]$IsAdmin) "$(if ($IsAdmin) {'Administrator: '})$(if ($GitStatus) {"posh~git ~ $($GitStatus.RepoName) [$($GitStatus.Branch)] ~ "})PowerShell $($PSVersionTable.PSVersion) ($PID)"}
 
     [PoshGitTextSpan]$DefaultPromptPrefix       = ''
-    [PoshGitTextSpan]$DefaultPromptSuffix       = '$(''>'' * ($nestedPromptLevel + 1)) '
-    [PoshGitTextSpan]$DefaultPromptDebugSuffix  = ' [DBG]$(''>'' * ($nestedPromptLevel + 1)) '
-
-    [bool]$DefaultPromptEnableTiming            = $false
-    [PoshGitCellColor]$DefaultPromptTimingColor = [PoshGitCellColor]::new()
-
+    [PoshGitTextSpan]$DefaultPromptSuffix       = '$(">" * ($nestedPromptLevel + 1)) '
+    [PoshGitTextSpan]$DefaultPromptDebug        = ' [DBG]:'
     [PoshGitTextSpan]$DefaultPromptPath         = '$(Get-PromptPath)'
+
+    [bool]$DefaultPromptWriteStatusFirst        = $false
     [bool]$DefaultPromptAbbreviateHomeDirectory = $false
 
+    [bool]$DefaultPromptEnableTiming            = $false
+    [PoshGitTextSpan]$DefaultPromptTimingFormat = ' {0}ms'
```

Comment:
I don't envision too many folks using prompt timing at all.  But for those of us that do, I prefer not to have the `ms` in there.  This is pretty easy way to accomplish this.